### ISSUE TITLE
Frame around spacemacs text (help, changelog)

### ISSUE THREAD

**Issue by jakub-gonet** [syl20bnr/spacemacs#10283]
#### Description :octocat:
Text frame around help (and others) isn't very fitting.

#### Reproduction guide :beetle:
- Start Emacs

*Observed behaviour:* :eyes: :broken_heart:
![image](https://user-images.githubusercontent.com/12465392/35609116-58c907da-065c-11e8-943c-f8fd632e1cac.png)
![image](https://user-images.githubusercontent.com/12465392/35609126-6be71ad2-065c-11e8-8622-133312b5c892.png)
![image](https://user-images.githubusercontent.com/12465392/35609128-6dc171ae-065c-11e8-8b76-116610ce5942.png)
![image](https://user-images.githubusercontent.com/12465392/35609158-8edbd10e-065c-11e8-85a3-8acb2a01ff93.png)

*Expected behaviour:* :heart: :smile:
Correct uppper and bottom line

#### System Info :computer:
- OS: gnu/linux Manjaro
- Emacs: 25.3.1
- Spacemacs: 0.200.13
- Spacemacs branch: master (rev. 582f9aa4)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm emacs-lisp)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GCONF GSETTINGS NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES

**Comment by duianto** [comment]
The frame is probably aligned to the recommended Source Code Pro font.

If you look at the "Warnings:" section in your third screenshot:

![](https://user-images.githubusercontent.com/12465392/35609128-6dc171ae-065c-11e8-8b76-116610ce5942.png)

It says:
>Cannot find any of the specified fonts (Source Code Pro)! Font settings may not
be correct

Which means that the Source Code Pro font isn't installed.

Here's the link to the font:
https://github.com/adobe-fonts/source-code-pro/releases/tag/2.030R-ro%2F1.050R-it

And here are the instructions for how to install fonts (if you need them):
https://github.com/adobe-fonts/source-code-pro#font-installation-instructions

**Comment by Alexander-Miller** [comment]
Note that you don't have to use Source Code Pro. Any font that looks good with these glyphs will do. Personally I've been using Fantasque Sans Mono since forever.

**Comment by jakub-gonet** [comment]
Yeah, I didn't connect these two things together. I'm closing issue in that case.

**Comment by duianto** [comment]
The font Alexander-Miller mentioned, seems to be this font: https://github.com/belluzj/fantasque-sans

@jakub-gonet Here are some instructions if you that want to try another font.
I downloaded: `FantasqueSansMono-Normal.zip`
from here: https://github.com/belluzj/fantasque-sans/releases
I then extracted and installed the `.ttf` fonts.
(In Windows I selected all 4 fonts, right clicked and chose `Install`)

In `.spacemacs`, I commented out "Source Code Pro", and added "Fantasque Sans Mono":
```
   dotspacemacs-default-font '(
                               ;; "Source Code Pro"
                               "Fantasque Sans Mono"
                               :size 13
                               :weight normal
                               :width normal)
```
After restarting Emacs, then the new font is shown.
